go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/DazWilkin/cloud-builders
```

Comment:
Is this file necessary? It should be enough to have the go.mod in `go/examples/multi_step/` right?

Reply:
Hmmm... user error.

Reply:
Apologies. Fixed: https://github.com/GoogleCloudPlatform/cloud-builders/pull/518/commits/ecb516fc9775eed1d5c9926c4a71054765569c3e